<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userlogined2</class>
 <widget class="QDialog" name="userlogined2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>445</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>539</x>
     <y>200</y>
     <width>208</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Change Password</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>New Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Confirm</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLineEdit" name="lineEdit_5"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_6"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>Submit Changes</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>201</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Update Information</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>First Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Last Name</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLineEdit" name="lineEdit_7"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_8"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="text">
       <string>Submit Changes</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <widget class="QLabel" name="label_19">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Welcome, </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>281</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <widget class="QPushButton" name="pushButton_7">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Book Flight</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_8">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Book Cruise</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>You have</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Points.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
